app/views/kaui/accounts/index.html.erb

```diff
@@ -42,12 +43,26 @@ $(document).ready(function() {
     "pageLength": <%= @limit %>,
     "displayStart": <%= @offset %>,
     <% unless @ordering.blank? %>
-        "order": [[ 0, "<%= @ordering %>" ]],
+        "order": [[ 1, "<%= @ordering %>" ]],
     <% end %>
     "processing": true,
     "serverSide": true,
     "search": {"search": "<%= @search_query %>"},
-    "ajax": "<%= accounts_pagination_path(:ordering => @ordering, :format => :json) %>"
+    "ajax": "<%= accounts_pagination_path(:ordering => @ordering, :format => :json) %>",
+    "rowCallback": function( row, data, index ) {
+      if ( data[0] == 1 ) {
+        $('td:eq(0)', row).html( '<span class="label label-info account-child-label">Child</span>' );
+      }else{
+        $('td:eq(0)', row).html('')
+      }
+    },
+    "columnDefs": [
+    { "orderable": false, "targets": 0, "createdCell": function (td, cellData, rowData, row, col) {
+        $(td).css('padding-left', '20px');
```

Comment:
Nit: might break responsiveness. Probably not worth fixing it right now though, as most screens would probably benefit from mobile testing again.